.github/workflows/swift-integration.yml

```
@@ -11,8 +11,8 @@ jobs:
     runs-on: macos-latest
     
     env:
-      # yorkie server v0.6.14
-      YORKIE_RB_URL: "https://raw.githubusercontent.com/Homebrew/homebrew-core/13fd4a1991db44f80170ff5c41ab98bd89565cf1/Formula/y/yorkie.rb"
+      # yorkie server v0.6.18
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 14: did not find expected key

(syntax-check)